public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/email.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Creates the Organizer Debrief Survey email in the Camptix email queue.
+ */
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Email;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Constants.
+ */
+const EMAIL_KEY_ID = 'organizer_debrief_survey_email';
+
+/**
+ * Return the email ID.
+ *
+ * @return mixed
+ */
+function get_email_id() {
+	return get_option( EMAIL_KEY_ID );
+}
+
+/**
+ * Return lead organizer email.
+ *
+ *  @return string
+ */
+function get_lead_organizer_email() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Email Address'][0];
+}
+
+/**
+ * Return lead organizer full name.
+ *
+ * @return string
+ */
+function get_lead_organizer_full_name() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Organizer Name'][0];
+}
+
+/**
+ * Return WordCamp location.
+ *
+ * @return string
+ */
+function get_wordcamp_location() {
+	$meta = get_wordcamp_post()->meta;
+	return $meta['Location'][0];
+}
+
+
+/**
+ * Return a token to put into the survey URL.
+ *
+ * @return string
+ */
+function get_encryption_token() {
```

Comment:
```suggestion
function get_verification_token() {
```

Technically this is [hashing, rather than encryption](https://www.thesslstore.com/blog/difference-encryption-hashing-salting/).

Reply:
Ah, yes, you're right. Messed up on the basics.. Thanks!